R/lump.R

```diff
@@ -38,20 +40,36 @@
 #' fct_lump(x, n = -3)
 #' fct_lump(x, prop = -0.1)
 #'
+#' # Use weighted frequencies
+#' w <- c(rep(2, 50), rep(1, 50))
+#' fct_lump(x, n = 5, weights = w)
+#'
 #' # Use ties.method to control how tied factors are collapsed
 #' fct_lump(x, n = 6)
 #' fct_lump(x, n = 6, ties.method = "max")
 #'
-fct_lump <- function(f, n, prop, other_level = "Other",
+fct_lump <- function(f, n, prop, weights, other_level = "Other",
```

Comment:
I realise now that `weights` should have a default value since it's option. @wilkox are you still interested in this PR? If not, I'm happy to make the change, but since you've put so much time into it, I thought you might like to carry it home.

Reply:
Still interested in seeing this merged 😃 Added a default `weights = NULL`, which seems less confusing than `weights = 1`.